locale/da_DK/common.xml

```diff
@@ -297,7 +297,7 @@
 	<message key="common.loading">Indlæser …</message>
 	<message key="debug.memoryUsage">Brug af hukommelse</message>
 	<message key="form.startsWith">begynder med</message>
-	<message key="navigation.dashboard">Instrumentpanel</message>
+	<message key="navigation.dashboard">Dashboard</message>
```

Comment:
I thought this might me an intentional change… I think it's hard to translate dashboard, and I might even have introduced it to the German translation as well.